apps/modernization-ui/src/apps/search/laboratory-report/labReportFormTypes.ts

```diff
@@ -23,6 +23,8 @@ type General = {
     enteredBy?: Selectable[];
     entryMethods?: Selectable[];
     eventDate?: EventDate;
+    eventId?: Identification['value'];
+    eventType?: Identification['type'];
```

Comment:
issue:  Please use the `identification` property.  The transformer is already set up to convert from the `Identification` type to the `LabReportEventId` type expected by the API.